api/sources/tests/extraction/test_hanze.py

```diff
@@ -81,3 +81,39 @@ def test_get_owners(self):
                 "name": "Johan de Acteur"
             }
         ])
+
+
+class TestPersonsExtraction(ExtractorTestCase):
+
+    source = "hanze"
+    entity = "persons"
+
+    def test_get_api_count(self):
+        self.assertEqual(self.extractor.get_api_count(self.extractor.data), 455)
+
+    def test_get_api_next_cursor(self):
+        self.assertEqual(self.extractor.get_api_next_cursor(self.extractor.data), "offset|10|10")
+
+    def test_get_api_previous_cursor(self):
+        self.assertIsNone(self.extractor.get_api_previous_cursor(self.extractor.data))
+
+    def test_get_name(self):
+        self.assertEqual(self.results[0]["name"], "Obi-Wan Kenobi")
+
+    def test_get_isni(self):
+        self.assertIsNone(
+            self.results[0]["isni"],
+            "Isni data is not available in the data."
+        )
+
+    def test_get_is_employed(self):
+        self.skipTest(
+            "According to Jasper Bedaux in an email on 6th March 2023 HvA won't pass along non-employed people"
```

Comment:
Hanze is niet HvA. Logica moet wel getest voor Hanze. Als het goed is komen True en False allebei voor in de data. Eventueel moeten we de fixture een beetje aanpassen om het zover te krijgen.

Reply:
Helemaal vergeten! Ik wou je hier nog over vragen. Klopt het ook dat er geen isni in de hanze bron zit?